app/assets/stylesheets/views/_report-child-abuse.scss

```diff
@@ -0,0 +1,8 @@
+.phone-numbers-report-child-abuse {
+  padding-top: 15px;
+  padding-bottom: 25px;
+
+  p.tel-report-child-abuse {
+    font-size: 25px;
```

Comment:
Can this use one of the govuk_frontend_toolkit core mixins?
Perhaps this one: https://github.com/alphagov/govuk_frontend_toolkit/blob/master/stylesheets/_typography.scss#L70

More details:
https://github.com/alphagov/govuk_frontend_toolkit/blob/master/docs/mixins.md#typography